src/Migration/Migration_Runner.php

```diff
@@ -40,10 +42,11 @@ public function run( $batch_size = 10 ) {
 	public function migrate_actions( array $action_ids ) {
 		do_action( 'action_scheduler/custom_tables/migration_batch_starting', $action_ids );
 
+		remove_all_actions( 'action_scheduler_stored_action' );
```

Comment:
Why remove all actions?

I think we only want to remove the _created date_ log entry, which could be done with something like:

```php
remove_action( 'action_scheduler_stored_action', array( ActionScheduler::logger(), 'log_stored_action', 10 ) );
```